pkpdapp/pkpdapp/templates/dataset_form.html

```diff
@@ -11,7 +11,7 @@
 
 <h2>Update Dataset {{dataset.name}}</h2>
 
-<form action="" method="post">
+<form action="" method="post"  enctype="multipart/form-data">
   {% csrf_token %}
```

Comment:
Can we please add some text to this page detailing the csv format required for upload? i.e. required columns